pkg/vpcmodel/abstractVPC.go

```diff
@@ -29,6 +39,7 @@ type Node interface {
 	IsPublicInternet() bool
 	Details() string
 	DetailsMap() map[string]string
+	Kind() string
```

Comment:
Consider moving `Details()`, `DetailsMap()` and `Kind()` to `NamedResourceIntf` (and maybe rename it to `VPCResourceIntf`)

Reply:
opened an issue to apply this in a separate PR